lib/vanagon/engine/base.rb

```diff
@@ -57,7 +57,7 @@ def ship_workdir(workdir)
 
       def retrieve_built_artifact
         FileUtils.mkdir_p("output")
-        Vanagon::Utilities.rsync_from("#{@remote_workdir}/output/*", "#{@target_user}@#{@target}", "output", @platform.ssh_port)
+        Vanagon::Utilities.rsync_from("#{@remote_workdir}/output/*", "#{@target_user}@#{@target}", "output/", @platform.ssh_port)
```

Comment:
I can't remember... I make this change a while ago. I assume I did run into an issue moving files to/from the windows box though https://github.com/puppetlabs/vanagon/commit/b36aefa54a3f9c84681ce72b3bdd61a144aa0041 is the commit